lib/index.coffee

```diff
@@ -15,7 +15,7 @@ dockerMtimeStream()
 	.on 'data', (layer_mtimes) ->
 		current_mtimes = layer_mtimes
 
-garbageCollect = (reclaimSpace) ->
+exports.garbageCollect = (reclaimSpace) ->
```

Comment:
It isn't, but I think that'd be a good thing to add. I'll add it as part of this PR actually :+1: